### DISCUSSION TITLE
Day 9! Better late than never

### DISCUSSION THREAD

**Discussion by CarolineMorton** [CarolineMorton/AdventOfCode_WIR#12] [Ideas]
I got slightly behind here - how are you all getting on?

**Comment by AR2202** [comment]
I've gone back to Haskell now. But might come back to Rust at some point.

**Reply by AR2202** [reply]
[Day 9](https://github.com/AR2202/AOC2024/blob/main/src/Day09.hs) in Haskell if anyone is interested. Got a bit messy though.
